src/presentation/cli/controller/ssl.go

```diff
@@ -0,0 +1,134 @@
+	"github.com/spf13/cobra"
+)
+
+type SslController struct {
+	sslService *service.SslService
+}
+
+func NewSslController(
+	persistentDbSvc *internalDbInfra.PersistentDatabaseService,
+	transientDbSvc *internalDbInfra.TransientDatabaseService,
+) *SslController {
+	return &SslController{
+		sslService: service.NewSslService(persistentDbSvc, transientDbSvc),
+	}
+}
+
+func (controller *SslController) Read() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "ReadSslPairs",
+		Run: func(cmd *cobra.Command, args []string) {
+			cliHelper.ServiceResponseWrapper(controller.sslService.Read())
+		},
+	}
+
+	return cmd
+}
+
+func (controller *SslController) Create() *cobra.Command {
+	var virtualHostsSlice []string
+	var certFilePathStr, keyFilePathStr string
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "CreateSslPair",
+		Run: func(cmd *cobra.Command, args []string) {
+			requestBody := map[string]interface{}{
+				"virtualHosts": virtualHostsSlice,
+			}
+
+			certFilePath, err := valueObject.NewUnixFilePath(certFilePathStr)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, "InvalidCertificateFilePath")
+			}
+			certContentStr, err := infraHelper.GetFileContent(certFilePath.String())
+			if err != nil {
+				cliHelper.ResponseWrapper(false, "OpenSslCertificateFileError")
+			}
+			requestBody["certificate"] = certContentStr
+
+			privateKeyFilePath, err := valueObject.NewUnixFilePath(keyFilePathStr)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, "InvalidSslPrivateKeyFilePath")
+			}
+			privateKeyContentStr, err := infraHelper.GetFileContent(
+				privateKeyFilePath.String(),
+			)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, "OpenSslPrivateKeyFileError")
+			}
+			requestBody["key"] = privateKeyContentStr
+
+			cliHelper.ServiceResponseWrapper(controller.sslService.Create(requestBody))
+		},
+	}
+
+	cmd.Flags().StringSliceVarP(
+		&virtualHostsSlice, "virtualHosts", "v", []string{}, "VirtualHosts",
+	)
+	cmd.MarkFlagRequired("virtualHosts")
+	cmd.Flags().StringVarP(
+		&certFilePathStr, "certFilePath", "c", "", "SslCertificateFilePath",
+	)
+	cmd.MarkFlagRequired("certFilePath")
+	cmd.Flags().StringVarP(&keyFilePathStr, "keyFilePath", "k", "", "SslKeyFilePath")
+	cmd.MarkFlagRequired("keyFilePath")
+	return cmd
+}
+
+func (controller *SslController) Delete() *cobra.Command {
+	var sslPairIdStr string
+
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "DeleteSslPair",
+		Run: func(cmd *cobra.Command, args []string) {
+			requestBody := map[string]interface{}{
+				"id": sslPairIdStr,
+			}
+
+			cliHelper.ServiceResponseWrapper(controller.sslService.Delete(requestBody))
+		},
+	}
+
+	cmd.Flags().StringVarP(&sslPairIdStr, "pairId", "i", "", "SslPairId")
+	cmd.MarkFlagRequired("pairId")
+	return cmd
+}
+
+func (controller *SslController) DeleteVhosts() *cobra.Command {
+	var sslPairIdStr string
+	var virtualHostsSlice []string
+
+	cmd := &cobra.Command{
+		Use:   "remove-vhosts",
```

Comment:
Is remove the standard we have been using on other controllers? I think we use delete all the time to comply with the CRUD acronym.

Reply:
Done!